src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/YourSingleApprenticeReadyForReviewStatus.cshtml

```diff
@@ -0,0 +1,66 @@
+﻿@using SFA.DAS.EmployerAccounts.Web.Extensions
+@using SFA.DAS.EmployerAccounts.Web.Helpers
+@model SFA.DAS.EmployerAccounts.Web.ViewModels.CohortV2ViewModel
```

Comment:
Please pass in just the ApprenticeshipViewModel rather than the CohortViewModel

Reply:
done